src/components/RandomBeer.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { useState, useEffect } from 'react'
+import Navbar from '../components/Navbar'
+import axios from 'axios';
+
+function RandomBeer() {
+    const [random, setRandom] = useState({});
+    const [oneBeerDet, setOneBeerDet] = useState({});
+
+    const getBeersApi = async () => {
+        try {
+            const response = await axios.get(`https://ih-beers-api2.herokuapp.com/beers/${random}`);
```

Comment:
This `API` already has a link that give you a random beer every time, so here the correct link is:
`'https://ih-beers-api2.herokuapp.com/beers/random'`